charts/node-analyzer/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: node-analyzer
 description: Sysdig Node Analyzer
 
 # currently matching Sysdig's appVersion 1.14.34
-version: 1.11.3
+version: 1.11.4
```

Comment:
It seems a behavioural change, I suggest to bump it as a minor, not a patch release.

Reply:
@mavimo thanks. done :)